### ISSUE TITLE
Staging Review finding: Error is not being focused on submit

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#122409]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Design System & Forms Library
**Product:** Form 21-4140
**Feature:** Employment Questionnaire

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** Yes
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Duplicate of known issue: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4993

<details><summary>Link, screenshot or steps to recreate</summary>

Safari/Voiceover issue.

<img width="573" height="605" alt="error not focused" src="https://github.com/user-attachments/assets/d371d238-599b-46f6-ae00-9730afd5d012" />

</details>

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 3.3.1 Error Identification - Level A](https://www.w3.org/TR/WCAG22/#error-identification)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
